### PR TITLE
create model for order service

### PR DESCRIPTION
**Pull request by LEHOANGGLAM**
create model for order service

### CODE REVIEW

File: order/src/main/java/com/yas/order/model/Order.java [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
+    private float discount;
+    private int numberItem;
+    private BigDecimal totalPrice;
+    private BigDecimal deliveryFee;
+    private EDeliveryMethod deliveryMethod;
+    private EDeliveryStatus deliveryStatus;
+    private EPaymentMethod paymentMethod;
+
+    @OneToMany(mappedBy = "orderId", fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+    Set<OrderDetail> orderDetails;
```
Comment (chanhnguyen6975) [inline order/src/main/java/com/yas/order/model/Order.java:38]: Consider initialize value in this line like Set<OrderDetail> orderDetails = new HashSet().

**Review by chanhnguyen6975 — APPROVED** [review]

File: order/src/main/java/com/yas/order/model/OrderDetail.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import java.math.BigDecimal;
+
+@Entity
+@Table(name = "order_detail")
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class OrderDetail {
```
Comment (thiennn) [inline order/src/main/java/com/yas/order/model/OrderDetail.java:14]: OrderItem seem a better name

Comment (LEHOANGGLAM) [reply order/src/main/java/com/yas/order/model/OrderDetail.java:14]: Thanks

File: order/src/main/java/com/yas/order/model/Order.java (hunk 1 above)
Comment (LEHOANGGLAM) [reply order/src/main/java/com/yas/order/model/Order.java:38]: Order always Order Item inside, thus, i dont think we need to initialize OrderItem(OrderDetail) when we initialize Order. we will set it on function "createOrder " in the later PR

**Review by DieuNguyenThi — APPROVED** [review]

**Pull request merged**